perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -71,6 +71,17 @@
 	 * @throw ResourceNotExistsException When Resource with <code>id</code> doesn't exist.
 	 */
 	/*#
+	 * Returns all non-empty Member, User, Member-Resource and User-Facility attributes for selected Member and Resource.
```

Comment:
I believe this should return only those named in ``attrNames``. and return all non-empty only for "empty list" of attrNames.